.all-contributorsrc

```diff
@@ -346,6 +346,16 @@
       "contributions": [
         "example"
       ]
+    },
+    {
+      "login": "motleydev",
```

Comment:
Let's make sure to add @karsasmus and @kairichard to this as reviewers as well. Thanks!